apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -179,6 +179,12 @@ ConversationView.prototype = {
     return this.composerAccessors.addAttachment();
   },
 
+  takeAttachmentScreenshot: function(index) {
```

Comment:
not completely sure of the name, as we may think it's for attachments in messages from the list... Ideally I'd like a Composer view :) I'm sure @azasypkin has something like this in his head already ;)

In the mean time, how about using "takeComposerAttachmentScreenshot" ? it's quite long but at least it's meaningful?